scala-package/infer/pom.xml

```diff
@@ -2,14 +2,15 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
     <parent>
         <artifactId>mxnet-parent_2.11</artifactId>
         <groupId>ml.dmlc.mxnet</groupId>
         <version>1.2.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
     </parent>
-    <modelVersion>4.0.0</modelVersion>
 
-    <artifactId>mxnet-infer</artifactId>
+    <artifactId>mxnet-infer_2.11</artifactId>
```

Comment:
What exactly is the "2.11" here? To me, it sounds like this is some kind of centralized version. Maybe use a variable instead? (also above). Is this related to "{scala.binary.version}"?

Reply:
It is the Scala binary version, this is how other modules in this package are built. We won't be refactoring the entire project here.